hardware/raspberrypi/bootmodes/nvme.md

```diff
@@ -1,38 +1,38 @@
 # NVMe SSD Boot (BETA)
 
-NVMe (Non Volatile Memory express) is a standard for accessing Solid State Drives (SSD) via a PCI express bus (PCIe). You can connect these drives via the PCIe slot on a Compute Module (CM) IO board allowing a CM4 to boot from SSD.
+NVMe (non-volatile memory express) is a standard for accessing solid state drives (SSDs) via a PCIe bus. You can connect these drives via the PCIe slot on a Compute Module 4 (CM4) IO board, allowing a CM4 to boot from SSD.
 
-## Hardware Needed
+## Required hardware
 
-You need an NVMe M.2 SSD. You can’t plug an M.2 SSD directly into the PCIe slot on the IO board so you’ll need an adapter. Be careful to get the right one. Search for PCI-E 3.0 x1 Lane to M.2 NGFF M-Key SSD Nvme PCI Express Adapter Card.
+You need an NVMe M.2 SSD. You cannot plug an M.2 SSD directly into the PCIe slot on the IO board - an adaptor is needed. Be careful to get the correct type: a suitable adaptor can be found online by searching for 'PCI-E 3.0 x1 Lane to M.2 NGFF M-Key SSD Nvme PCI Express Adapter Card'.
 
-You don't need it, but if you want to access an NVMe drive from a PC, you might want to get a USB adapter. Search for NVME PCI-E M-Key Solid State Drive External Enclosure.
-
-Linux supports NVMe drives so you should be able to plug it in to the IO board and see your NVMe device to check it works. Run `ls -l /dev/nvme*`.
+Linux supports accessing NVMe drives without modification. (Booting from NVMe requires a pre-release version of the Raspberry Pi Linux kernel - see "required software" below). To check that your NVMe drive is connected correctly, boot Linux from another drive and run `ls -l /dev/nvme*`; example output is shown below.
```

Comment:
Hmmm, I wonder if it's worth simply removing the sentence-in-brackets? :shrug:

Reply:
That was there to differentiate between OS support for NVMe once the OS has booted, and support in the OS for booting from NVMe. I've reworded it and changed `Linux` to `Raspberry Pi OS`, since that's what we're actually talking about. (Important to be specific, and correctly set expectations).